retailer-oas/retailer.yaml

```diff
@@ -404,7 +404,7 @@ paths:
       tags:
         - timeslots
       parameters:
-        - name: id
+        - name: deliveryAreaId
```

Comment:
I would suggest here to ask for the pickingLocationID instead. As this is the information they know and the info they have used to load the catalog on the website. Delivery areas should be something mostly internal on our logistics.